.github/workflows/macos.yaml

```diff
@@ -3,36 +3,42 @@ name: macos
 on: [push, pull_request]
 
 jobs:
-
   macos-build:
-    runs-on: ${{ matrix.macos-version }}
+    runs-on: ${{ matrix.os.os }}
     strategy:
       fail-fast: false
       matrix:
-        macos-version:
-          - 'macos-latest'
+        os:
+          - {arch: clang_64, os: macos-13}
```

Comment:
It looks like you are no longer using the arch variable. Would you like to return to a simple list of macOS versions?